From d26db2d41045ce6412ebf541de66744868d4af6b Mon Sep 17 00:00:00 2001
From: Jack Olivieri <jolivieri@anaconda.com>
Date: Fri, 20 Dec 2024 12:13:07 +0100
Subject: [PATCH] update paths

---
 setup.py | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index 3c33297..548bb5c 100644
--- a/setup.py
+++ b/setup.py
@@ -52,11 +52,14 @@ shims_enabled = not cflags_defined
 # obvious.
 if shims_enabled and not qpdf_source_tree:
     if 'bsd' in sys.platform:
-        shim_includes = ['/usr/local/include']
+        #shim_includes = ['/usr/local/include']
+        shim_includes = [environ.get('PREFIX')+'include']
         shim_libdirs = []
     elif 'darwin' in sys.platform and machine() == 'arm64':
-        shim_includes = ['/opt/homebrew/include', '/opt/local/include']
-        shim_libdirs = ['/opt/homebrew/lib', '/opt/local/lib']
+        #shim_includes = ['/opt/homebrew/include', '/opt/local/include']
+        #shim_libdirs = ['/opt/homebrew/lib', '/opt/local/lib']
+        shim_includes = [environ.get('PREFIX')+'include']
+        shim_libdirs = [environ.get('PREFIX')+'lib']
     else:
         shim_includes = []
         shim_libdirs = []
-- 
2.39.5 (Apple Git-154)

